config/config.schema.json

```diff
@@ -426,6 +426,34 @@
         "cert"
       ]
     },
+    "mise":{
+      "properties": {
+        "deploy" :{
+          "type": "boolean"
+        },
+        "azureAdInstance":{
+          "type":"string"
+        },
+        "validAppId0":{
+          "type":"string",
+          "description": ""
+        },
+        "validAppId1":{
+          "type":"string",
+          "description": ""
+        },
```

Comment:
i wonder what apps we are going to feed in here.
potentially it could make sense to properly name them so people have an easier time when they read the config

Reply:
Yeah, it's not clear to me what those apps will be named.  Just came up with a name that correlates well to the MISE environment variables.

Reply:
do we at least know what these apps represent? are this apps we create or apps provided MSFT?
if we know about the purpose of both apps, we can name the fields appropriately.

just a nit. for now we can merge and come back later once we know more about the apps

Reply:
@katherinelc321 do you have suggestions for naming this config?

Reply:
one appID should represent ARM which we don't have to create and the other appID should be the RP/frontend one which we need to create